### DISCUSSION TITLE
meetup request

### DISCUSSION THREAD

**Discussion by ahadnawaz585** [Zain-ul-din/LGU-BetterTimeTable#23] [Q&A] [answered]
i am also a student in lgu in 3 semester working as a PEAN stack developer want to meet u if u still in uni to have a healthy motivating talk about how u get to collab with other members and what consequences u faced and want to know how it been !

**Comment by Zain-ul-din** [comment, marked as answer]
Hey, Thanks for reaching out! send your WhatsApp number on this mail : `fa20-bscs-147@lgu.edu.pk` also you can follow my  [timetable](https://www.lgutimetable.live/timetable/7th%20Semester%20Fall-2023%20-%20Fall-2020%20BSCS%20Sec%20E) I will be in the class from 8 to 11 tomorrow and on Friday i'll attend all lectures.

> how u get to collab with other members and what consequences u faced and want to know how it been !

i plan to write documentation for this project but i don't have time due to other projects and job. With the project documentation i also plan to write history of this project like when and why we created this project?.

**Reply by Asadullah-byte** [reply]
Hi zain ul din, I am a 3rd year at lgu we are conducting arbisoft mock interviews at University. It would be great if you could participate requirment s are OOP, DSA, DB  7/8sem students. Only 8 student get to participate 3are left contact me at fa-21-bsse-231@lgu.edu.pk

**Reply by Zain-ul-din** [reply]
@Asadullah-byte Date?

**Reply by Asadullah-byte** [reply]
24th of April by thrusday between 10am to 12pm
